examples/asset_licences.md

```diff
@@ -0,0 +1,9 @@
+The "Tuffy" font is public domain, was made by Thatcher Ulrich, and was accessed from here: http://www.publicdomainfiles.com/show_file.php?id=13486218041168
+The loop doodle_pop.ogg is public domain and was made by MrZeusTheCoder.
+opengl/resources/background.png is public domain, was made by Nidhoggn, and was accessed from here: https://opengameart.org/content/backgrounds-3
+shader/resources/background.jpg is public domain, was taken by Arcana Dea, and was accessed from here: https://www.publicdomainpictures.net/en/view-image.php?image=10979&picture=monarch-butterfly
+shader/resources/devices.png is public domain, was made by Kenny, and was accessed from here: https://www.kenney.nl/assets/game-icons
+sound/resources/ding.flac is public domain, was made by Kenny, and was accessed from here: https://www.kenney.nl/assets/interface-sounds
+sound/resources/killdeer.wav is public domain, was recorded by the US National Park Services, and was accessed from here: https://www.nps.gov/subjects/sound/sounds-killdeer.htm
+tennis/resources/ball.wav is public domain and was made by MrZeusTheCoder.
```

Comment:
Again, we'd need a source to link to, otherwise we end up with the same situation as we have right now 😄